packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -99,14 +99,21 @@ contract Bridge is EssentialContract, IBridge {
             });
     }
 
-    function enableDestChain(
```

Comment:
Jeff, if anyone can enable a destination chain, then somes users without much knowledge will believe the dest chain has a Bridge deployment for withdrawal, then they will try to send funds to that chain. I think we can simply fix the issue by adding `onlyOwner` to this funciton.

Reply:
Actually I have a better idea. Let me show it to you.

Reply:
<img width="1638" alt="Screenshot 2023-01-10 at 11 36 45" src="https://user-images.githubusercontent.com/99078276/211456414-06a123fa-7902-4f61-bccd-e9fb28843b80.png">

Then we can remove `state.destChains` and the `enableDestChain` function.

Reply:
OK Great. Since you have this done already, you can commit that in PR and we can close this one?